src/h5web/visualizations/heatmap/ColorBar.tsx

```diff
@@ -28,12 +33,10 @@ function ColorBar(props: Props): JSX.Element {
       />
       <svg className={styles.colorBarAxis} height={gradientHeight} width="2em">
         <AxisRight
-          scale={scaleLinear()
-            .domain(dataBounds)
-            .range([gradientHeight, 0])
-            .nice()}
-          strokeWidth={0}
+          scale={scale}
```

Comment:
Rather than a `linearScale`, the scale of the axis is now the `colorScale` (linear or log) mapped to the height of the gradient bar.